pkg/sql/rename_test.go

```diff
@@ -184,15 +184,21 @@ CREATE TABLE test.t (a INT PRIMARY KEY);
 	}
 
 	// Concurrently, rename the table.
-	threadDone := make(chan interface{})
+	threadDone := make(chan error, 1)
```

Comment:
no need to buffer this, right? you're always reading from the channel at the end of the test.